tests/test_generic_transformer.py

```diff
@@ -133,6 +133,7 @@ def test_handle_meta_deps(self, spec, no_meta_runtime_dep, no_meta_buildtime_dep
         ('%build\ntest\n\n', '%build\n{0}test\n{1}'.format(scl_enable, scl_disable)),
         ('%prep\ntest\ntest\n\n', '%prep\n{0}test\ntest\n{1}'.format(scl_enable, scl_disable)),
         ('%check\ntest \\\ntest\n\n', '%check\n{0}test \\\ntest\n{1}'.format(scl_enable, scl_disable)),
+        ('%install\ntest\n\n%postun\ntrue\n\n', '%install\n{0}test\n{1}\n\n%postun\n{2}true\n{3}'.format(scl_enable, scl_disable, scl_enable, scl_disable)),
```

Comment:
Could you please please use the numeric index in `format` method and change this line to:
```
('%install\ntest\n\n%postun\ntrue\n\n', '%install\n{0}test\n{1}\n\n%postun\n{0}true\n{1}'.format(scl_enable, scl_disable)),
```
It works the same way, just without duplicating the args to `format`.

Reply:
Aha, will do -- python is not my native language :-)